src/components/pages/Desk/DeskConfirmDialog/DeskConfirmDialog.jsx

```diff
@@ -0,0 +1,47 @@
+import PropTypes from "prop-types"
+import React from 'react'
+
+import { ReactComponent as AlertSvg } from "../../../../icons/ico-alert-grey.svg"
```

Comment:
`icons` à un alias dans la configuration webpack

```suggestion
import { ReactComponent as AlertSvg } from "icons/ico-alert-grey.svg"
```